mitodl/mitxonline
# Fixed dashboard card spacing and image sizing

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #167

#### What's this PR do?
- Fixes the dashboard UI so that the items appear as cards in the mobile view, and the feature images are rendered correctly
- Adds suggested aspect ratio for course feature images in help text

#### How should this be manually tested?
- Enroll in at least one course
- Navigate to the dashboard

In mobile and desktop views, the course cards on the dashboard should have the correct image sizing, and the card should have margins on both sides in the mobile view.

Also try changing the course page's feature image to an image with a different aspect ratio and make sure it is still rendered correctly in mobile and desktop

#### Any background context you want to provide?
[Desktop design](https://projects.invisionapp.com/d/main#/console/21535972/455535008/preview)
[Mobile design](https://projects.invisionapp.com/d/main#/console/21535972/455498563/preview)

The suggested aspect ratio that I added was based on the largest rendering of the feature image, which is on the [home page](https://projects.invisionapp.com/d/main#/console/21535972/455419669/preview)

#### Screenshots (if appropriate)
The different images below have different aspect ratios for demonstration purposes

![ss 2021-09-23 at 09 41 38 ](https://user-images.githubusercontent.com/14932219/134530683-be860670-cfb6-49ab-ae1c-1699bf2398b4.png)
![ss 2021-09-23 at 09 42 02 ](https://user-images.githubusercontent.com/14932219/134530690-1ac69c53-89f5-4531-8b14-bef08afbbb98.png)
![ss 2021-09-23 at 09 46 42 ](https://user-images.githubusercontent.com/14932219/134530694-72bb2cc4-e49d-425a-808d-d3bdf4e09102.png)
![ss 2021-09-23 at 09 46 50 ](https://user-images.githubusercontent.com/14932219/134530696-a0524545-42f5-4b61-97c4-3f0c9190281c.png)
